crds/tests/test_config.py

```diff
@@ -23,6 +27,49 @@
 TEST_MAPPATH = os.path.join(CRDS_TESTING_CACHE, "mappings")
 TEST_TEMP_DIR = tempfile.mkdtemp(prefix='crds-test-')
 
+
+def generate_test_data():
+    import asdf
+    from roman_datamodels import datamodels as rdd
+    from roman_datamodels.testing import factories
+
+    reftype = {
+        'flat': factories.create_flat_ref,
+        'dark': factories.create_dark_ref,
+        'gain': factories.create_gain_ref,
```

Comment:
gain, mask, and readnoise don't have all three of the arrays, so this function will fail if we try to create files of those types

Reply:
i know, its just there as a reference so that I remember to generate each type.